assets/prototype/domain/shared/dateRangeInput/DateRangeInput.js

```diff
@@ -0,0 +1,53 @@
+import { useState } from 'react';
+import isFunction from 'lodash/isFunction';
+import moment from 'moment';
+import { DateRangeInput as DefaultDateRangeInput } from '@blueprintjs/datetime';
+import { FORMATS } from './formatSelect';
+import { MomentDateRange } from './momentDate';
+
+import './style.css';
+
+const DateRangeInput = ({ endDate, onFieldUpdate, startDate }) => {
+	const format = FORMATS[0];
+	const defaultRangeValues = [startDate || null, endDate || null];
+	const [range, setRange] = useState(defaultRangeValues);
+
+	const handleRangeChange = (value) => {
+		const [start, end] = value;
+		const isUpdatable = isFunction(onFieldUpdate) && start && end;
+
+		if (isUpdatable) {
+			const format = 'YYYY-MM-DD HH:mm:ss';
```

Comment:
would make a nice global constant...

I'm thinking we should add some kind of ConfigProvider to the top level app contexts,
which could include things like the various date and time formats we need, as well as timezone information, currency, etc

but that's for another PR :)

Reply:
Sure, great suggestion.
This variable is a temp solution for prototype.
But for the long-term I've borrowed `formatSelect.js` from blueprint and will centralize that into a context provider as you said. I wasn't sure about the context provider at first, but later I recalled that we might need to pull some user specific information from WP.
Great suggestion and thanks for tracking this into that project card.